test/org/redhatcop/util/files/params/jenkins-s2i.params

```diff
@@ -0,0 +1,3 @@
+JENKINS_GIT_URL=https://github.com/etsauer/pipeline-library.git
+JENKINS_GIT_BRANCH=shared-lib
```

Comment:
I think we should remove this line since the default is `master` which will be true once the PR is merged.

Reply:
yes